app/helpers/payment_details_helper.rb

```diff
@@ -68,4 +68,8 @@ def stripe_billing_address_json
     }.to_json
   end
 
+  def get_country_code(billing_address_data)
+    merchant_countries.select {|c| c.include? billing_address_data.country}.first[1]
```

Comment:
This can raise `NilError` when the country is not included in the list (for example the `billing_address_data.country == nil`)

Reply:
@josemigallas  any suggestion to improve?

Reply:
Why @josemigallas ? :smile: 

```suggestion
    label, _code = merchant_countries.find {|label, code| label == billing_address_data.country.to_s.downcase }
    code
```

Still this is suboptimal...

Reply:
Then maybe use safe navigation? Not sure if it's syntactically correct:
```ruby
merchant_countries.select({|c| c.include? billing_address_data.country})&.first[1]
```

Or just split in two steps:
```ruby
countries = merchant_countries.select {|c| c.include? billing_address_data.country}
countries.first[1] if countries.present?
```